test/fixtures/mson-mixin-nonexistent.json

```diff
@@ -0,0 +1,109 @@
+{
+  "_version": "3.0",
+  "metadata": [],
+  "name": "<API name>",
+  "description": "<API description>\n\n",
+  "element": "category",
+  "resourceGroups": [],
```

Comment:
Question: is there a reason to not include this kind of data in a root-level element?